tianshou/policy/psrl/psrl.py

```diff
@@ -0,0 +1,182 @@
+import torch
+import numpy as np
+from typing import Dict, List, Union, Optional
+import mdptoolbox
+
+from tianshou.policy import BasePolicy
+from tianshou.data import Batch, ReplayBuffer, to_torch_as
+
+
+class PSRLModel(object):
+    """Implementation of Posterior Sampling Reinforcement Learning Model.
```

Comment:
Could you please add a reference here? Also in the top of `README.md` and `docs/index.rst`.